pom.xml

```
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.jenkins-ci.plugins</groupId>
     <artifactId>plugin</artifactId>
-    <version>4.70</version>
+    <version>4.74</version>
```

Latest release of plugin parent pom.
```suggestion
    <version>4.76</version>
```